docs/workflows.rst

```diff
@@ -44,29 +55,114 @@ warp to the MNI space.
 .. figure:: _static/segmentation.svg
     :scale: 100%
 
-    Segmentation (FAST).
+    Tissue segmentation (FAST).
```

Comment:
I you want to be completely exact, then this would be *Brain tissue segmentation*, but tissue segmentation is OK.